apps/settings/js/panels/languages/languages.js

```diff
@@ -56,7 +56,7 @@ define(function(require) {
     },
     showMoreLanguages: function() {
       SettingsCache.getSettings(function(result) {
-        var version = result['deviceinfo.os'];
+        var version = result['langpack.channel'];
```

Comment:
where's the value comes from? will it be provided by gecko, or should we preset a value for it?